src/app/zap-templates/zcl/data-model/chip/webrtc-requestor-cluster.xml

```diff
@@ -27,11 +27,14 @@ Git: 0.7-summer-2025-5-g06c4d5596
     <name>WebRTC Transport Requestor</name>
     <code>0x0554</code>
     <define>WEB_RTC_TRANSPORT_REQUESTOR_CLUSTER</define>
-    <description>The WebRTC transport requestor cluster provides a way for stream consumers (e.g. Matter Stream Viewer) to establish a WebRTC connection with a stream provider.</description>
+    <description/>
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
     <globalAttribute code="0xFFFD" side="either" value="1"/>
-    <attribute code="0x0000" side="server" define="CURRENT_SESSIONS" type="array" entryType="WebRTCSessionStruct" default="0">CurrentSessions</attribute>
+    <attribute code="0x0000" side="server" define="CURRENT_SESSIONS" type="array" entryType="WebRTCSessionStruct" default="0">
+      <description>CurrentSessions</description>
+      <access op="read" privilege="administer"/>
```

Comment:
So it's admin on requestor but manage on provider?  Is that asymmetry intended?

Reply:
That is intended. The data at the Requestor is more sensitive and so Fabric Admins on the Requestor fabric are allowed to access it. The camera side(Provider) is slightly more open, allowing a Requestor to tell its user who else is using the camera(provider) if it is out of resources.